quora-service/src/main/java/com/upgrad/quora/service/business/AdminBusinessService.java

```diff
@@ -72,4 +74,30 @@ public UserEntity logoutUser(final String authorizationToken) throws SignOutRest
 
         return userAuthEntity.getUser();
     }
+
+    public void deleteUser(final String uuid, final String authorizationToken) throws UserNotFoundException, AuthorizationFailedException {
```

Comment:
Exception handler not added for `UserNotFoundException`. Please add it in `com.upgrad.quora.api.exception.RestExceptionHandler` class.